tern/analyze/default/default_common.py

```diff
@@ -14,11 +14,14 @@
 
 from tern.classes.notice import Notice
 from tern.report import errors
+from tern.report import report
 from tern.utils import constants
 from tern.utils import rootfs
 from tern.utils import general
+from tern.analyze import common
 from tern.analyze.default.command_lib import command_lib
 from tern.analyze.default import filter as fltr
+from tern.analyze.default.container import image as cimage
```

Comment:
This import is causing a lot of circular import linking errors. Ideally, you'd want to follow the code structure described here: https://github.com/tern-tools/tern/blob/main/docs/navigating-the-code.md

So things in `container` should import from `default_common` and not the other way around.

Reply:
So @nishakm  should I remove this function `default_common_procedure` from `default_common.py` to `common.py`  ?

Reply:
No you will need to remove this line.

Reply:
If I remove this line, then how `cimage.analyze` will be used inside that `default_common.py`? Is there any way around for this?